term_webassembly.go

```diff
@@ -0,0 +1,66 @@
+//go:build js
+// +build js
+
+package term
+
+import "io"
+import "os"
+import "errors"
+
+// terminalState holds the platform-specific state / console mode for the terminal.
+type terminalState struct{}
+
+// GetWinsize returns the window size based on the specified file descriptor.
+func getWinsize(fd uintptr) (*Winsize, error) {
+	return getWinsize(fd)
+}
+
+func isTerminal(fd uintptr) bool {
+	return true
+}
+
+func saveState(fd uintptr) (*State, error) {
+	return &State{}, nil
+}
+
+func stdStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
+	return os.Stdin, os.Stdout, os.Stderr
+}
+
+func disableEcho(fd uintptr, state *State) error {
+	return nil
+}
```

Comment:
This seems rather dangerous, or at least it means that it will compile, but silently discard setting this option and (e.g.) print passwords?

Wondering if it should either error, or if we need to make it a compile option when trying to use these parts with webassembly (i.e., don't have these functions implemented).


cc @AkihiroSuda @vvoland any thoughts?